test/SlowTests/Issues/RavenDB-21273.cs

```diff
@@ -42,7 +42,10 @@ public async Task ExceptionWhenImportingAdditionalAssembliesWithCommunityLicense
             var file = GetTempFileName();
             try
             {
-                using (var store = GetDocumentStore())
+                using (var store = GetDocumentStore(new Options()
+                       {
+                           IgnoreDisabledDatabase = true
```

Comment:
Can you elaborate on that change? I see it was applied to many tests

Reply:
its a typo I wanted to use `IgnoreDocumentCompression` since those test are using community